README.md

```diff
@@ -0,0 +1,53 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is a project for microverse 2.0 training
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://test.com)
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+
+
+## Authors
+
+👤 **Peter Robert**
+
+- Github: [@peterrobert](https://github.com/peterrobert)
+- Twitter: [@ptahwambui93](https://twitter.com/Ptahwambui93)
```

Comment:
- [ ] Kindly put a valid link to your Twitter account because the current one isn't showing an existing profile.
This can leave a bad impression on future viewers. So, it's important to make sure that everything is valid.